d3a_api_client/utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+import requests
+import json
+import logging
+
+
+def retrieve_jwt_key_from_server(domain_name):
+    resp = requests.post(
+        f"{domain_name}/api-token-auth/",
+        data=json.dumps({"username": os.environ["API_CLIENT_USERNAME"],
+                         "password": os.environ["API_CLIENT_PASSWORD"]}),
+        headers={"Content-Type": "application/json"})
+    if resp.status_code != 200:
+        logging.error(f"Request for token authentication failed with status code {resp.status_code}."
+                      f"Response body: {resp.text}")
+        return
```

Comment:
```suggestion
        return None
```
?

Reply:
It is exactly the same as now actually, return value of a function that just returns is None